### ISSUE TITLE
Feature/unfeature an event from events list in Backend (starring/unstarring) fails for non-admins

### ISSUE THREAD

**Issue by ameotoko** [contao/contao#2505]
**Affected version(s)**

Contao 4.10.4

**Description**

Feature/unfeature an event from events list in Backend (starring/unstarring) fails for non-admins with `Contao\CoreBundle\Exception\AccessDeniedException: Invalid command "feature".`

```
Contao\CoreBundle\Exception\AccessDeniedException:
Invalid command "feature".

  at var/cache/dev/contao/dca/tl_calendar_events.php:681
  at tl_calendar_events->checkPermission()
     (var/cache/dev/contao/dca/tl_calendar_events.php:1145)
  at tl_calendar_events->toggleFeatured('168', true)
     (vendor/contao/core-bundle/src/Resources/contao/classes/Ajax.php:401)
  at Contao\Ajax->executePostActions(object(DC_Multilingual))
     (vendor/contao/core-bundle/src/Resources/contao/classes/Backend.php:425)
  at Contao\Backend->getBackendModule('calendar', null)
     (vendor/contao/core-bundle/src/Resources/contao/controllers/BackendMain.php:167)
  at Contao\BackendMain->run()
     (vendor/contao/core-bundle/src/Controller/BackendController.php:48)
  at Contao\CoreBundle\Controller\BackendController->mainAction()
     (vendor/symfony/http-kernel/HttpKernel.php:158)
  at Symfony\Component\HttpKernel\HttpKernel->handleRaw(object(Request), 1)
     (vendor/symfony/http-kernel/HttpKernel.php:80)
  at Symfony\Component\HttpKernel\HttpKernel->handle(object(Request), 1, true)
     (vendor/symfony/http-kernel/Kernel.php:201)
  at Symfony\Component\HttpKernel\Kernel->handle(object(Request))
     (web/index.php:31)
```

**Comment by ameotoko** [comment]
The bug is due to a missing line:

https://github.com/contao/contao/blob/7e6bacd4e334bdd1a32996dab12d3b05e1e161cc/calendar-bundle/src/Resources/contao/dca/tl_calendar_events.php#L640-L646

```diff
    case 'toggle':
+   case 'feature':

**Comment by ausi** [comment]
Closed in favor of #2524
